2014-01-14 00:46:47,448 [main] ERROR  partnerIntg.App - UUID for the Job is : -6147565202388270079
2014-01-14 00:46:48,517 [main] ERROR  partnerIntg.App - Exception in main function
java.io.FileNotFoundException: c:\goodtech\batch\xmlIntg.properties (No such file or directory)
	at java.io.FileInputStream.open(Native Method) ~[na:1.7.0_40]
	at java.io.FileInputStream.<init>(FileInputStream.java:146) ~[na:1.7.0_40]
	at partnerIntg.App.main(App.java:87) ~[classes/:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.7.0_40]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57) ~[na:1.7.0_40]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.7.0_40]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:1.7.0_40]
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:120) [idea_rt.jar:na]
2014-01-14 00:50:16,463 [main] ERROR  partnerIntg.App - UUID for the Job is : 7136218237175939784
2014-01-14 00:50:17,127 [main] ERROR  partnerIntg.App - Exception in main function
java.io.FileNotFoundException: c:\goodtech\batch\xmlIntg.properties (No such file or directory)
	at java.io.FileInputStream.open(Native Method) ~[na:1.7.0_40]
	at java.io.FileInputStream.<init>(FileInputStream.java:146) ~[na:1.7.0_40]
	at partnerIntg.App.main(App.java:87) ~[classes/:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.7.0_40]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57) ~[na:1.7.0_40]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.7.0_40]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:1.7.0_40]
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:120) [idea_rt.jar:na]
2014-01-14 00:51:08,653 [main] ERROR  partnerIntg.App - UUID for the Job is : 854654988075551175
2014-01-14 01:09:02,209 [main] ERROR  partnerIntg.App - UUID for the Job is : 956658733850181643
2014-01-14 01:09:03,004 [main] ERROR  partnerIntg.App - Error in XML validation
org.xml.sax.SAXParseException: cvc-complex-type.2.4.a: Invalid content was found starting with element 'BillToCustomer'. One of '{SoldToCustomer}' is expected.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:198) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:134) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:437) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:368) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:325) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator$XSIErrorReporter.reportError(XMLSchemaValidator.java:458) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator.reportSchemaError(XMLSchemaValidator.java:3237) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator.handleStartElement(XMLSchemaValidator.java:1796) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator.startElement(XMLSchemaValidator.java:746) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:376) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2768) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:606) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:116) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:846) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:775) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.jaxp.validation.StreamValidatorHelper.validate(StreamValidatorHelper.java:151) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.jaxp.validation.ValidatorImpl.validate(ValidatorImpl.java:116) ~[na:1.7.0_40]
	at javax.xml.validation.Validator.validate(Validator.java:124) ~[na:1.7.0_40]
	at partnerIntg.App.validateAgainstXSD(App.java:52) [classes/:na]
	at partnerIntg.App.main(App.java:94) [classes/:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.7.0_40]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57) ~[na:1.7.0_40]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.7.0_40]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:1.7.0_40]
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:120) [idea_rt.jar:na]
2014-01-14 01:14:19,431 [main] ERROR  partnerIntg.App - UUID for the Job is : 4614442642631246899
